Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
 
-gem "github-pages", "112" #Update me once in a while: https://github.com/github/pages-gem/releases
+gem "github-pages", "147" #Update me once in a while: https://github.com/github/pages-gem/releases
```

Comment:
Should this one be in `docs/docker.github.io:docs-builder` as well? Looks like it's tied to the version of github-pages.

Actually; looking at the Dockerfile for [starefossen/github-pages](https://hub.docker.com/r/starefossen/github-pages/), We may not even need it at all: https://github.com/Starefossen/docker-github-pages/blob/147/Dockerfile#L6-L16

Reply:
You need the gem for when it's running in a random environment, like when someone just does `bundle exec jekyll serve` from the checked-out repo, or for Netlify. In fact, the Netlify builds on the older branches were failing because there was no Gemfile (now they are failing for other reasons I can't figure out). The Gemfile that lives in the source is different from the Gemfile imported from the github-pages Docker image (and probably should be ignored by Docker for that reason, but that is a call for a different chunk of work).

Reply:
Ahhhhh, right, yes that makes sense 👍